bee-message/src/output/mod.rs

```diff
@@ -101,7 +101,7 @@ impl Output {
         }
     }
 
-    ///
+    /// Return the IOTA amount of an `Output`.
```

Comment:
```suggestion
    /// Returns the amount of an `Output`.
```

Can also be Shimmer amount